xlive/XLive/xnet/upnp.cpp

```diff
@@ -50,14 +50,14 @@ void ForwardPorts()
 	ModuleUPnP upnp;
 	Utils::UPnPResult upnpResult(Utils::UPnPErrorType::None, 0);
 
-	upnpResult = upnp.UPnPForwardPort(false, H2Config_base_port, H2Config_base_port, "Halo2");
+	upnpResult = upnp.UPnPForwardPort(false, H2Config_base_port, H2Config_base_port, "Halo2_port");
 	LOG_INFO_NETWORK("ForwardPorts() - Halo2 port forwarding result: {}", upnpResult.ErrorCode);
 
-	upnpResult = upnp.UPnPForwardPort(false, (H2Config_base_port + 1), (H2Config_base_port + 1), "Halo2_1");
-	LOG_INFO_NETWORK("ForwardPorts() - Halo2_1 port forwarding result: {}", upnpResult.ErrorCode);
+	upnpResult = upnp.UPnPForwardPort(false, H2Config_base_port + 1, H2Config_base_port + 1, "Halo2_port1");
+	LOG_INFO_NETWORK("ForwardPorts() - Halo2 port forwarding result: {}", upnpResult.ErrorCode);
 
-	upnpResult = upnp.UPnPForwardPort(true, (H2Config_base_port + 10), (H2Config_base_port + 10), "Halo2_QoS");
-	LOG_INFO_NETWORK("ForwardPorts() - Halo2_QoSport forwarding result: {}", upnpResult.ErrorCode);
+	upnpResult = upnp.UPnPForwardPort(true, H2Config_base_port + 10, H2Config_base_port + 10, "Halo2_port_QoS");
```

Comment:
Should probably make the values of 1 and 10 here constants across the project. ie `H2Config_base_port` + k_QoS_port_offset` or something similar